.github/workflows/CICD.yml

```diff
@@ -455,12 +458,12 @@ jobs:
       shell: bash
       run: |
         make build
-    - name: "`make test`"
+    - name: "`make nextest`"
       shell: bash
-      run: |
-        make test
+      run: make nextest CARGOFLAGS="--profile ci --hide-progress-bar"
       env:
         RUST_BACKTRACE: "1"
+        CARGO_TERM_COLOR: "always"
```

Comment:
@sylvestre you previously objected to this change. Have you changed your mind?

In any case we can specify it just once I believe.

Reply:
yes, I changed my mind :)
intermittent should not happen in the coreutils in general

Reply:
yeah, i missed that change
could you please remove it?

Reply:
i am looking at logs in github action too